Gemfile

```diff
@@ -43,6 +43,8 @@ end
 group :development do
   gem 'better_errors'
   gem 'binding_of_caller'
+  gem 'wraith', '~> 3.0'
+  gem 'phantomjs', '~> 2.0'
```

Comment:
i don't use the vm, and not everyone does, we shoudln't be coding for the vm imo